ex04.ex

```diff
@@ -74,11 +74,21 @@ defmodule Ex04 do
   helps you do that. And, if you use that function, what does it return? That
   return value will be the thing you have to manipulate.
   """
+  
```

Comment:
# +1 for the use of function arguments
# -1 for processing the input list twice 

:)